KIPs/kip-7-fungible_token.md

```diff
@@ -83,6 +150,18 @@ These methods are optional.
 |[addPauser](#addminter)|function addPauser(address _account) public |
 |[renouncePauser](#renounceminter)|function renouncePauser() public |
 
+Every [KIP-7](https://klaytn.github.io/kips/KIPs/kip-7-fungible_token) compliant contract with Pause methods MUST implement [KIP-13](https://klaytn.github.io/kips/KIPs/kip-interface_query_standard) `supportsInterface` with the identifier `0xXXXXXXXX` for Pause methods.
+```solidity
+/// @title KIP-721 Non-Fungible Token Standard, optional pause extension
+///  Note: the KIP-13 identifier for this interface is 0xXXXXXXXX.
+interface IKIP7Pause {
+    function supportsInterface(bytes4 interfaceID) external view returns (bool);    // KIP-13 interface
+
+    function transfer(address to, uint256 value) public whenNotPaused returns (bool);
```

Comment:
The example of IKIP7Pause methods should be changed.

Reply:
I fixed. :) Thanks.